<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLTestsAppClass</class>
 <widget class="QMainWindow" name="GLTestsAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRPMapEditorQt</string>
  </property>
  <widget class="CGLWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>States</string>
    </property>
    <addaction name="actionBase_State"/>
    <addaction name="actionTest_Shader"/>
    <addaction name="actionTest_CArm"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShader_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShader_Editor">
   <property name="text">
    <string>Shader Editor</string>
   </property>
  </action>
  <action name="actionBase_State">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Base State</string>
   </property>
  </action>
  <action name="actionTest_Shader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test Shader</string>
   </property>
  </action>
  <action name="actionTest_CArm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test CArm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGLWidget</class>
   <extends>QWidget</extends>
   <header>GLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>ActionTriggered(QAction*)</slot>
 </slots>
</ui>
